papaparse.js

```diff
@@ -58,6 +58,7 @@
 	Papa.BAD_DELIMITERS = ['\r', '\n', '"', Papa.BYTE_ORDER_MARK];
 	Papa.WORKERS_SUPPORTED = !IS_WORKER && !!global.Worker;
 	Papa.SCRIPT_PATH = null;	// Must be set by your code if you use workers and this lib is loaded asynchronously
+	Papa.NODE_STREAM_INPUT = {};
```

Comment:
I think an empty object is too generic for this constant. Probably a integer (which is unexpected input) will be a better value.

Reply:
Would `Papa.NODE_STREAM_INPUT = 1` be okay?  My thought with the object is to have a constant that could not be produced outside of the library so there would be no chance that someone would accidentally invoke the node specific streaming.  If someone from a browser were to invoke node streaming an error would be thrown because there is a dependency on the `stream` module provided by node core libs.

Reply:
The constant is a good idea, I prefer to be explicit in this case instead of allowing null values and then having unexpected results. 

1 seems a good value to me.